tests/gnrc_ipv6_ext_frag/main.c

```diff
@@ -617,6 +612,31 @@ static int shell_test_cmd(int argc, char **argv)
     return 0;
 }
 
+static int send_test_pkt(int argc, char **argv)
+{
+    if (argc != 1) {
```

Comment:
Here and everywhere throughout the tests: Do we really need a usage line for a no-arg command?

Reply:
Not really, I did it as an alternative to casting to void. I can remove if you prefer.

Reply:
> Not really, I did it as an alternative to casting to void. I can remove if you prefer.

Yeah, I think I prefer casting to void here. It's not much, but it safes some bytes of code size and memory but it could be the deciding factor if the test fits a board or not.

Reply:
Ok, can I squash right away?

Reply:
Yepp.

Reply:
I said you may squash right away ;-)